src/main/kotlin/lotto/domain/Prize.kt

```diff
@@ -0,0 +1,24 @@
+package lotto.domain
+
+/**
+ * 클래스에 대한 설명을 적어주세요.
+ * Created by Jaesungchi on 2022.05.25..
+ */
+
+enum class Prize(val matchCount: Int, val reward: Int) {
+    LOSER(0, 0),
+    FIFTH(3, 5000),
+    FOURTH(4, 500000),
+    THIRD(5, 1500000),
+    SECOND(5, 30000000),
+    FIRST(6, 2000000000)
```

Comment:
```suggestion
    LOSER(0, 0),
    FIFTH(3, 5_000),
    FOURTH(4, 500_000),
    THIRD(5, 1_500_000),
    SECOND(5, 30_000_000),
    FIRST(6, 2_000_000_000)
```

이렇게 하면 더 가독성 있어 보일 것 같아요!